### PR TITLE
Update examples for CLI commands

### PR DESCRIPTION
**Pull request by anderseknert**
* And fix a typo in IR docs

Signed-off-by: Anders Eknert <anders@eknert.com>

### CODE REVIEW

File: cmd/eval.go [hunk 1]
```diff
@@ -168,11 +168,11 @@ Examples
 
 To evaluate a simple query:
 
-    $ opa eval 'x = 1; y = 2; x < y'
+    $ opa eval 'x := 1; y := 2; x < y'
 
 To evaluate a query against JSON data:
 
-    $ opa eval --data data.json 'data.names[_] = name'
+    $ opa eval --data data.json 'name := data.names[_]'
```
Comment (srenatus) [inline cmd/eval.go:175]: Alternatively, but a bit of a mouthful:
```
opa eval --import future.keywords.in --data data.json "some name in data.names"
```

File: docs/content/ir.md [hunk 2]
```diff
@@ -454,4 +454,4 @@ example.
 The following examples show how the test suite is used internally:
 
 * [`github.com/open-policy-agent/opa/topdown#TestRego`](https://github.com/open-policy-agent/opa/blob/main/topdown/exported_test.go)
-* [`github.com/open-policy-agent/opa/test/wasm/cmd/wasm-rego-testgen`](https://github.com/open-policy-agent/opa/blob/main/test/wasm/cmd/wasm-rego-testgen/main.go)
\ No newline at end of file
+* [`github.com/open-policy-agent/opa/test/wasm/cmd/wasm-rego-testgen`](https://github.com/open-policy-agent/opa/blob/main/test/wasm/cmd/wasm-rego-testgen/main.go)
```
Comment (srenatus) [inline docs/content/ir.md:457]: 🤔 We could remove that link -- it's using a different set of test files: https://github.com/open-policy-agent/opa/tree/main/test/wasm/assets

**Review by srenatus — APPROVED** [review]

File: cmd/eval.go (hunk 1 above)
Comment (anderseknert) [reply cmd/eval.go:175]: Yeah, I considered it, but I agree that it's a bit much for a potentially first encounter of the eval command.

File: docs/content/ir.md (hunk 2 above)
Comment (anderseknert) [reply docs/content/ir.md:457]: Not sure I follow. Do we not test the Wasm implementation against the Yaml test suite?

Comment (srenatus) [reply docs/content/ir.md:457]: Yep but in another place. These tests are different

Comment (anderseknert) [reply docs/content/ir.md:457]: Alright, should we link to that location instead?

Comment (srenatus) [reply docs/content/ir.md:457]: Yep, that would be here: https://sourcegraph.com/github.com/open-policy-agent/opa@0f912fcbe2206deb3f3fe2e164f76821098cd0d0/-/blob/internal/wasm/sdk/test/e2e/external_test.go?L57

Comment (anderseknert) [reply docs/content/ir.md]: Fixed, thanks!

**Review by srenatus — APPROVED** [review]

**Pull request merged**